scripts/Tools/Makefile

```diff
@@ -604,6 +607,29 @@ $(FV3CORE_LIBDIR)/libfv3core.a: $(EXEROOT)/FMS/libfms.a
 
 dyn_grid.o: $(FV3CORE_LIBDIR)/libfv3core.a
 endif
+
+ifdef MPAS_LIBDIR
```

Comment:
Could this be done in the cam buildlib and libmpas.a combined into libatm.a so that no changes are needed in the cime Makefile?
The following will combine liba.a and libb.a into libab.a
ar crsT libab.a liba.a libb.a

Reply:
I'll let @brian-eaton answer this.

Reply:
We were following the example of the COSP and FV3 library builds.  Not sure how this is different.

I've looked in buildlib and don't see how it would be done from there.  There are interdependencies between the mpas lib and the cam build which are expressed in the Makefile.  libmpas depends on cam_abortutils.o, and dyn_comp.o depends on libmpas.  So building libmpas is synchronized between the builds of cam_abortutils.o and dyn_comp.o.  Note that this is also true for the COSP library build: cospsimulatro_intr.o depends on libcosp which depends on cam_abortutils.o.